products/oceanic-current-fronts-world-ocean-circulation-AGLHS-HR/collection.json

```diff
@@ -0,0 +1,104 @@
+{
+    "type": "Collection",
+    "id": "oceanic-current-fronts-world-ocean-circulation-AGHLS-HR",
+    "stac_version": "1.0.0",
+    "description": "Fronts derived from high resolution remote sensing SST observations by SEVIRI L3C from OSISAF over Agulhas region. The spatial resolution of the product is 0.05 deg, the temporal resolution is 1 hour and it it projected in WGS 84 (EPSG:4326)",
+    "links": [
+      {
+        "rel": "root",
+        "href": "../../catalog.json",
+        "type": "application/json",
+        "title": "Open Science Catalog"
+      },
+      {
+        "rel": "via",
+        "href": "https://www.worldoceancirculation.org/Products#/metadata/6c776c43-425b-4d29-9934-0822696f15d8",
+        "title": "Access"
+      },
+      {
+        "rel": "via",
+        "href": "https://drive.google.com/file/d/1GIxRA1xJ5I-ch7aXSqh9I_y-YGBe3xfM/view?usp=sharing",
```

Comment:
is this link going to be maintained in Google Drive? Is there some reference to that document from Project's website instead?